tools/ete/homology_classifier.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="homology_classifier" name="Homology Classifier and Filter" version="@VERSION@.1">
+    <description>from a genetree utilising the ETE Toolkit</description>
+    <macros>
+        <import>ete_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <stdio>
+        <!-- Anything other than zero is an error -->
+        <exit_code range="1:" />
+    </stdio>
+    <command>
```

Comment:
You can add ` detect_errors="exit_code"` to `<command>` and remove the `<stdio>` section.